README.md

```diff
@@ -74,7 +74,12 @@ make --directory=$(pwd)/buildroot O=$(pwd)/${DEFCONFIG}_output savedefconfig # s
 ```
 
 ### kernel
-TODO
+```sh
+. env.sh
+make --directory=$(pwd)/buildroot O=$(pwd)/${DEFCONFIG}_output linux-menuconfig
+make --directory=$(pwd)/buildroot O=$(pwd)/${DEFCONFIG}_output linux-savedefconfig
+make --directory=$(pwd)/buildroot O=$(pwd)/${DEFCONFIG}_output linux-update-defconfig
```

Comment:
huh weird this has 2 steps (`*-savedefconfig` and `*-update-defconfig`) vs. the buildroot single `savedefconfig`.

Reply:
makes me wonder if we're missing a step for the regular buildroot config.

Reply:
very strange yes, one goes from `.config` -> `defconfig` in the linux build dir, the other copies it to the buildroot custom location

Reply:
ahh ok there's two levels of redirection. nice, this seems great for version controlling kconfigs.